Src/libCZI/CziUtils.cpp

```diff
@@ -111,73 +111,74 @@ using namespace libCZI;
 /// <returns>	A boolean indicating whether the coordinates are equal or not (according to the above definition of equality). </returns>
 /*static*/bool CziUtils::CompareCoordinate(const libCZI::IDimCoordinate* coord1, const libCZI::IDimCoordinate* coord2)
 {
-		bool areEqual = true;
-		CziUtils::EnumAllCoordinateDimensions(
-				[&](libCZI::DimensionIndex dim)->bool
-				{
-						int coordinate1, coordinate2;
-						if (coord1->TryGetPosition(dim, &coordinate1) == true)
-						{
-								if (coord2->TryGetPosition(dim, &coordinate2) != true)
-								{
-										areEqual = false;
-										return false;
-								}
-
-								if (coordinate1 != coordinate2)
-								{
-										areEqual = false;
-										return false;
-								}
-						}
-
-						return true;
-				});
-
-		return areEqual;
+    bool areEqual = true;
+    CziUtils::EnumAllCoordinateDimensions(
+        [&](libCZI::DimensionIndex dim)->bool
+        {
+            int coordinate1, coordinate2;
+            if (coord1->TryGetPosition(dim, &coordinate1) == true)
+            {
+                if (coord2->TryGetPosition(dim, &coordinate2) != true)
+                {
+                    areEqual = false;
+                    return false;
+                }
+
+                if (coordinate1 != coordinate2)
+                {
+                    areEqual = false;
+                    return false;
+                }
+            }
+
+            return true;
+        });
+
+    return areEqual;
 }
 
 /*static*/void CziUtils::EnumAllCoordinateDimensions(std::function<bool(libCZI::DimensionIndex)> func)
 {
-		static const DimensionIndex dims[] = { DimensionIndex::Z,DimensionIndex::C,DimensionIndex::T,DimensionIndex::R,DimensionIndex::S,
-												DimensionIndex::I,DimensionIndex::H,DimensionIndex::V,DimensionIndex::B };
-
-		for (int i = 0; i < sizeof(dims) / sizeof(dims[0]); ++i)
-		{
-				if (func(dims[i]) == false)
-						break;
-		}
+    static const DimensionIndex dims[] = { DimensionIndex::Z,DimensionIndex::C,DimensionIndex::T,DimensionIndex::R,DimensionIndex::S,
+                                            DimensionIndex::I,DimensionIndex::H,DimensionIndex::V,DimensionIndex::B };
+
+    for (int i = 0; i < sizeof(dims) / sizeof(dims[0]); ++i)
+    {
+        if (func(dims[i]) == false)
+        {
```

Comment:
This looks like some massive flaw/bug in the past -> not having those brackets there before seems to kind of change logic completely, doesn't it?

Reply:
Not sure what you mean... Here I added curly brackets around the break-statement
![image](https://user-images.githubusercontent.com/4881321/216309342-3d9313cc-b5de-443c-8d25-7bd2a378a73c.png)

This does not change the behavior here. But granted, it is not a formatting change (and maybe should not have gone in this PR). This is a coding-standard thing, the rule I am following here is "have braces always, also for single nested statements".  Those kind of "rules" need to go into documentation I guess - next PR then.